### ISSUE TITLE
Docs !!

### ISSUE THREAD

**Issue by peteringram0** [Ramotion/folding-cell#4]
This seems really nice but for someone new to swift the documents are really hard to follow.

**Comment by ober01** [comment]
any ideas how to improve it?

**Comment by peteringram0** [comment]
Would be nice to see a lot more screenshots per each section and maybe a video of how to implement?

**Comment by ober01** [comment]
add video adding constraints https://vimeo.com/154954299 to readme.md
I hope this help

**Comment by fboulegue** [comment]
to bad i did try to follow it, but failed hard :( the documentation is really really hard for "no experts" simply the complete setup is missing - would be great if someone could add a better documentation

**Comment by WikipediaBrown** [comment]
It would be really cool if you actually added documentation for the programmatic example and if you used technical language to describe what you are doing would help. It seems like somebody that don't know how to build projects in Xcode, Swift or Obj-C wrote the documentation.

**Comment by WikipediaBrown** [comment]
Also, none of these projects build and run. None of them can be rebuilt and even rewriting the code rom scratch doesn't work. I'm sure the library works itself but there's no way to tell that because there is no documentation on how to use the library and there are no working examples extant.

**Comment by WikipediaBrown** [comment]
Also, when advertising a programmatic solution, you should actually use a programmatic solution. Your programmatic example uses storyboards so its pretty much as useful as the storyboard example.

**Comment by ober01** [comment]
Follow this steps: https://github.com/Ramotion/folding-cell#usage
and look at this example: https://github.com/Ramotion/folding-cell/tree/master/FoldingCell/FoldingCell-Demo

In my opinion, this enough, if not maybe your level is too low
